proto/slinky/mm2/v1/params.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package slinky.mm2.v1;
+
+option go_package = "github.com/skip-mev/slinky/x/mm2/types";
+
+// Params defines the parameters for the x/marketmap module.
+message Params {
+  // MarketAuthorities is the list of authority accounts that are able to
+  // control updating the marketmap.
+  repeated string market_authorities = 1;
+
+  // Version is the schema version for the MarketMap data structure and query
+  // response.
+  uint64 version = 2;
```

Comment:
We're putting the version in params? I thought we'd be putting it in the market-map? How else would we be correlating between the mm data-structure + the version?

Reply:
Can we have this be a string which is essentially just the tag?

Reply:
The marketmap scheme is likely not going to change very often but we should have a way of signaling validators when to update their side-car with a new tag/version. imo you kinda accomplish both in the same case, any schema change to the market map is going require a new side car version anyways

Reply:
im assuming once this is ready we just set the default to v1.0.0